code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -192,11 +201,15 @@ Please contact me on #coderbus IRC. ~Carnie x
 
 /mob/living/carbon/human/update_inv_wear_id()
 	remove_overlay(ID_LAYER)
+
 	if(wear_id)
 		if(client && hud_used && hud_used.hud_shown)
 			wear_id.screen_loc = ui_id
 			client.screen += wear_id
 
+		if(dna && istype(dna.species, /datum/species/pony))

```

Comment:
There is no fucking way Ponies are going to be used outside of a meme event or April Fools (at best).

There's a difference between a Lizardperson, who at least has a lot of precedent (Books and Tabletop games) for existing but also somewhat fits in with the universe, and a bunch of cartoon ponies.